packages/components/src/docs/Migration/ui-image.mdx

```diff
@@ -0,0 +1,127 @@
+# Migration Guide: `ui-image`
+
+The ui-image has been transformed to `sd-media` and `sd-copyright`.
+
+# ui-image to sd-media
+
+Instead of mainly providing content via attributes, the new `sd-media` component uses slots to allow for more flexibility and customization.
```

Comment:
sd-media and sd-copyright are not using slot, they now are classes which can be used in HTML. Perhaps you could orient yourself according to the sd-table 🤔